lxd/instance_post.go

```diff
@@ -260,8 +260,18 @@ func instancePost(d *Daemon, r *http.Request) response.Response {
 
 		// If no member was selected yet, pick the member with the least number of instances.
 		if targetMemberInfo == nil {
+			var filteredCandidateMembers []db.NodeInfo
+
+			// The instance might already be placed on the node with least number of instances.
```

Comment:
Shouldnt the existing node be removed from the candidate list more generally, so that its removed from the candidate list when using scriptlets too, rather than just in the case of using last instances placement?